opcua/gopcua/sub.go

```diff
@@ -53,16 +53,16 @@ func (b client) Subscribe(cfg opcua.Config) error {
 	}
 
 	c := opcuaGopcua.NewClient(ep.EndpointURL, opts...)
-	if errC := c.Connect(b.ctx); err != nil {
-		b.logger.Error(errC.Error())
+	if errc := c.Connect(b.ctx); errc != nil {
```

Comment:
Why not just follow err propagation? Why new var errc and errs?

Reply:
Indeed, `err` is a special var in Go that can be re-assigned with `:=`